src/components/cluster/detail/WorkerNodesAzure.js

```diff
@@ -21,21 +22,30 @@ export const LineDiv = styled.div`
 `;
 
 function WorkerNodesAzure({ instanceType, nodes, showScalingModal }) {
+  const instanceTypeText = instanceType
+    ? `${instanceType.cpu_cores} CPUs, ${instanceType.memory_size_gb.toFixed(
```

Comment:
I just realised azure's instance type structure looks different to this, so we get a error on `toFixed`

In azure it's like `instanceType.memoryInMb`, which you then have to divide by 1000.

And `instanceType.numberOfCores` for the CPU core count.

Reply:
So it would look like this

```
  // prettier-ignore
  const instanceTypeText = instanceType
    ? `${instanceType.numberOfCores} CPUs, ${(instanceType.memoryInMb/1000).toFixed(0)} GB RAM`
    : '0 CPUs, 0 GB RAM';
```

Reply:
Didn't notice, thanks!
( this one and kvm views are the only views without tests.. )